### DISCUSSION TITLE
How can we import external DDL

### DISCUSSION THREAD

**Discussion by roccho-dev** [dineug/erd-editor#338] [Q&A]
How can we import external DDL?
I've checked [here](https://github.com/dineug/erd-editor/blob/master/packages/sql-ddl-parser/src/SQL_DDL_Test_Case.md)
And I've tried to enter example DDL in this page into DDL like this image

![image](https://github.com/dineug/erd-editor/assets/40359643/5e8266c1-93cc-4be1-a621-39eeb93f70cb)

Am I wrong anything?

**Comment by dineug** [comment]
In the erd area, in the context menu

**Reply by gostrek** [reply]
The file must have .sql extension. Only this one works, right? I've tried .ddl and .txt with SQL code but 'Just import the sql file'. Thanks btw!
